src/modules/authentication/components/Login/Login.tsx

```diff
@@ -1,7 +1,121 @@
+import styles from "./Login.module.css";
+import imgLogo from "./../../../../assets/images/PMSLogo.svg";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import { AUTH_URL, axiosInstanceUrl } from "../../../../services/EndPoints";
+import { useState } from "react";
+
+interface loginForm {
+  email: string;
+  password: string;
+}
 export default function Login() {
+  const navigate = useNavigate()
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+  } = useForm<loginForm>();
+
+  const onSubmit = async (data: loginForm) => {
+    try {
+      const response = await axiosInstanceUrl.post(AUTH_URL.LOGIN, data);
+      console.log(response);
+      navigate("/dashboard")
+    } catch (error) {
+      console.log(error);
```

Comment:
We need to add a toast message indicating the error to the user.

Reply:
this we do not have the toast installed to the project that what made me do it lik this

Reply:
We need to consider it to avoid having the user unable to proceed and unable to know the reason for the form not working.